ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -779,7 +775,7 @@ def test_chainexpiration_check_without_cert(self, move_ipa_ca_crt):
             assert check["result"] == "ERROR"
             assert check["kw"]["key"] == paths.IPA_CA_CRT
             assert check["kw"]["error"] == error_text
-            assert check["kw"]["msg"] == msg_text
+            assert msg_text in check["kw"]["msg"]
```

Comment:
This line should not be changed upstream, please see https://github.com/freeipa/freeipa-healthcheck/commit/1188d5beaf4a0fdfe98608d9259532b860dabeb1